jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -429,15 +425,17 @@ public JaegerSpan start() {
         }
       }
 
-      JaegerSpan jaegerSpan = new JaegerSpan(
-              JaegerTracer.this,
-              operationName,
-              context,
-              startTimeMicroseconds,
-              startTimeNanoTicks,
-              computeDurationViaNanoTicks,
-              tags,
-              references);
+      JaegerSpan jaegerSpan =
```

Comment:
Just curious, why not `JaegerSpan jaegerSpan = getObjectFactory().createSpan(` on one line? 
Would be less code changes overall.

Reply:
This was auto-formatted. I will try your style and see what the auto-formatter does with it.

Reply:
Didn't let me use this format. Changed it for me.